### PR TITLE
adding cloudformation template to source control

### PR DESCRIPTION
**Pull request by mhines-usgs**
analytics_data_cloudformation file has not been run yet but the resources already exist in aws, so I'll ask Ivan how to rectify that. we will try to use the ctek pipeline to test and deploy it. more details in JIRA.

### CODE REVIEW

File: analytics_data_cloudformation.json [hunk 1]
```diff
@@ -0,0 +1,397 @@
+{
```
Comment (wdwatkins) [inline analytics_data_cloudformation.json:1]: 👍

File: analytics_logs_cloudformation.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
+{
```
Comment (wdwatkins) [inline analytics_logs_cloudformation.json:1]: Isn't this one already checked in to the NWIS_Analytics repo?

Comment (wdwatkins) [reply analytics_logs_cloudformation.json:1]: I do think this one might live in some kind of pipeline repo someday after a repo refactor, but should probably stick with the code that generates the data for now.

Comment (mhines-usgs) [reply analytics_logs_cloudformation.json:1]: ahh I couldn't remember! thanks, I'll remove it if I can figure out how.

**Comment by wdwatkins** [comment]
The other one looks fine, but I guess we can hold off on merging if this is likely going to live in the Ctek space.

**Pull request closed without merging**